service-worker/prefetch-video/index.html

```diff
@@ -7,8 +7,9 @@
 
 <video controls  poster="static/poster.jpg"
   style="margin: 0 0 1em 0; max-width: 100%; width: 480px;">
-  <source src="static/gbike.webm" type="video/webm" />
-  <source src="static/gbike.mp4" type="video/mp4" />
+  <!-- Videos are stored remotely with CORS enabled -->
```

Comment:
We may want to specify that only `https://googlechrome.github.io` has been allowed in CORS Headers to not confuse developers who may try to it this URL and wondering why it doesn't work for them.